api/v2/views/volume.py

```diff
@@ -104,6 +103,9 @@ def perform_create(self, serializer):
             return inactive_provider(pna)
         except VOLUME_EXCEPTIONS as e:
             raise exceptions.ParseError(detail=e.message)
+        except Exception as exc:
+            logger.exception("Error occurred creating a v2 volume")
```

Comment:
Since we're starting to send logs to `lumberjack`, should be include some information that might track the `Exception` back to an instance-alias, volume-alias, or something that would get us back to a person having a problem?